gui/src/i18n/de/translation.json

```diff
@@ -0,0 +1,462 @@
+{
+  "websocket": {
+    "connecting": "Verbindung zum Server wird hergestellt",
+    "connection-lost": "Verbindung zum Server verloren. Versuche Verbindung wiederherzustellen ..."
+  },
+  "tips": {
+    "find-tracker": "Sie sind sich nicht sicher, welcher Tracker welcher ist? Schütteln Sie einen Tracker, um das zugehörige Element hervorzuheben.",
+    "do-not-move-heels": "Stellen Sie sicher, dass Ihre Absätze während der Aufnahme nicht bewegt werden!"
+  },
+  "body-part": {
+    "NONE": "Nicht zugewiesen",
+    "HEAD": "Kopf",
+    "NECK": "Hals",
+    "RIGHT_SHOULDER": "Rechte Schulter",
+    "RIGHT_UPPER_ARM": "Rechter Oberarm",
+    "RIGHT_LOWER_ARM": "Rechter Unterarm",
+    "RIGHT_HAND": "Rechte Hand",
+    "RIGHT_UPPER_LEG": "Rechtes oberes Bein",
+    "RIGHT_LOWER_LEG": "Rechtes unteres Bein",
+    "RIGHT_FOOT": "Rechter Fuß",
+    "CHEST": "Brust",
+    "WAIST": "Taille",
+    "HIP": "Hüfte",
+    "LEFT_SHOULDER": "Linke Schulter",
+    "LEFT_UPPER_ARM": "Linker Oberarm",
+    "LEFT_LOWER_ARM": "Linker Unterarm",
+    "LEFT_HAND": "Linke Hand",
+    "LEFT_UPPER_LEG": "Linkes oberes Bein",
+    "LEFT_LOWER_LEG": "Linkes unteres Bein",
+    "LEFT_FOOT": "Linker Fuß"
+  },
+  "skeleton-bone": {
+    "NONE": "Keine",
+    "HEAD": "Kopfverschiebung",
+    "NECK": "Halslänge",
+    "TORSO": "Rumpflänge",
+    "CHEST": "Brustabstand",
+    "WAIST": "Tailleabstand",
+    "HIP_OFFSET": "Hüftversatz",
+    "HIPS_WIDTH": "Hüftbreite",
+    "LEGS_LENGTH": "Beinlänge",
+    "KNEE_HEIGHT": "Kniehöhe",
+    "FOOT_LENGTH": "Fußlänge",
+    "FOOT_SHIFT": "Fußverschiebung",
+    "SKELETON_OFFSET": "Skelettversatz",
+    "CONTROLLER_DISTANCE_Z": "Controllerabstand Z",
+    "CONTROLLER_DISTANCE_Y": "Controller Entfernung Y",
+    "FOREARM_LENGTH": "Unterarm Entfernung",
+    "SHOULDERS_DISTANCE": "Schulter Entfernung",
+    "SHOULDERS_WIDTH": "Schulter Breite",
+    "UPPER_ARM_LENGTH": "Oberarm Länge",
+    "ELBOW_OFFSET": "Ellbogen Versatz"
```

Comment:
Not sure if these should be written as one word or not, but I believe they should be

Reply:
Does not really seem to matter if it's one word or not but I'll change it to one